lib/kubernetes-deploy/kubernetes_resource/custom_resource_definition.rb

```diff
@@ -2,8 +2,86 @@
 module KubernetesDeploy
   class CustomResourceDefinition < KubernetesResource
     TIMEOUT = 2.minutes
+    ROLLOUT_CONFIG_ANNOTATION = "kubernetes-deploy.shopify.io/monitor-instance-rollout"
     GLOBAL = true
 
+    class RolloutConfig
```

Comment:
Let's stick this in its own file.

Reply:
Looking around I'm not sure I'd like to move this elsewhere, as it belongs squarely as a nested class of CustomResourceDefinition, even if it does bloat the file a little bit. Having it all in one place seems more advantageous to me in the end. If someone comes in with a tie-breaker I'll respect that decision, though